HW_6/task_1_2.py

```diff
@@ -0,0 +1,53 @@
+from random import sample
+
+BOARD_SIZE = 8
+
+
+def get_random_pos():
+    pos = sample([(x, y) for x in range(1, BOARD_SIZE + 1) for y in range(1, BOARD_SIZE + 1)], BOARD_SIZE)
+    return pos
+
+
+def check_queens_disposition(pos):
+    correct = True
+    for i in range(len(pos)):
+        for j in range(i + 1, len(pos)):
+            if pos[i][0] == pos[j][0] or pos[i][1] == pos[j][1] \
+                    or abs(pos[i][1] - pos[j][1]) == abs(pos[i][0] - pos[j][0]):
+                correct = False
+                break
+        if not correct:
+            break
+    if correct:
+        return True
+    else:
+        return False
```

Comment:
Можно значительно проще
```suggestion
    for i, (x0, y0) in enumerate(pos, 1):
        for x1, y1 in pos[i:]:
            if x0 == x1 or y0 == y1 or abs(x0 - x1) == abs(y0 - y1):
                return False

    return True
```